### ISSUE TITLE
Backpropagation

### ISSUE THREAD

**Issue by Hima-Mehta** [jostmey/rwa#8]
Hello, Earlier I had come across the recursive back propagation equations being used in RWA Now it seems to be unavailable.Could you please share it ?

**Comment by jostmey** [comment]
It is in the old version of the manuscript: https://arxiv.org/abs/1703.01253v2

I removed it from the updated manuscript because I want to double check the equations. The idea may be flawed. If it works, I will put the equations back in. TensorFlow will use normal backpropagation to the fit the model without needing the recursive gradient equations

**Comment by Hima-Mehta** [comment]
Ok Thanks.That solves the issue.

**Comment by jostmey** [comment]
Just to follow up on the issue of computing the gradients by a recursive forward pass. I did not have the equations written correctly in that earlier version of the manuscript but I think the concept will work. I am playing around with the idea some more, and hope to publish a follow up manuscript if the idea works,
